centralized-sampling-tests/sample-apps/golang-http-server/application.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/contrib/propagators/aws/xray"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	sample "go.opentelemetry.io/contrib/samplers/aws/xray"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/trace"
+	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"google.golang.org/grpc"
+)
+
+func getSampledSpanCount(name string, totalSpans string, attributes []attribute.KeyValue) int {
+	tracer := otel.Tracer(name)
+
+	var sampleCount = 0
+	totalSamples, _ := strconv.Atoi(totalSpans)
+	ctx := context.Background()
+
+	for i := 0; i < totalSamples; i++ {
+		_, span := tracer.Start(ctx, name, oteltrace.WithSpanKind(oteltrace.SpanKindServer), oteltrace.WithAttributes(attributes...))
+
+		if span.SpanContext().IsSampled() {
+			sampleCount++
+		}
+
+		span.End()
+	}
+
+	return sampleCount
+}
+
+func webServer() {
+	http.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		_, _ = w.Write([]byte("healthcheck"))
```

Comment:
```suggestion
		_, _ = w.Write([]byte("healthcheck"))
```
Is this returning an error? If so do not ignore it. At minimum log it.